libs/trustchain/tests/scenarios/tokenExpires.ts

```diff
@@ -1,17 +1,18 @@
 import Transport from "@ledgerhq/hw-transport";
 import { ScenarioOptions } from "../test-helpers/types";
+import { HWDeviceProvider } from "../../src/HWDeviceProvider";
+import { SDK } from "../../src/sdk";
 
-export async function scenario(
-  transport: Transport,
-  { pauseRecorder, sdkForName }: ScenarioOptions,
-) {
-  const sdk = sdkForName("Foo");
+export async function scenario(transport: Transport, { pauseRecorder }: ScenarioOptions) {
+  const hwDeviceProvider = new HWDeviceProvider();
+  const applicationId = 16;
```

Comment:
Where does it come from? Can we use a const shared if it's reused somewhere?

Reply:
It comes from this file when I started working on this refactor: https://github.com/LedgerHQ/ledger-live/blob/bf28296e92939103a6dd94fb17e09fe16ff6ba8b/libs/trustchain/src/test-scenarios/tokenExpires.ts#L6
When I rebased I changed it back because the new `sdkForName` helper doesn't allow setting a custom `HWDeviceProvider`. As it's just a mock value I thought it was fine to leave it as is.